packages/utils/ast-utils.js

```diff
@@ -429,6 +429,62 @@ function addProperty(j, p, key, value) {
 	}
 	return;
 }
+
+/**
+ *
+ * Recursively updates an object/property to a node
+ * @param {any} j — jscodeshift API
+ * @param {Node} p - AST node
+ * @param {String} key - key of a key/val object
+ * @param {Any} value - Any type of object
+ * @returns {Node} - the updated ast
+ */
+
+function updateProperty(j, p, key, value) {
+	let valForNode;
+	if (!p) {
+		return;
+	}
+	if (Array.isArray(value)) {
+		let arrExp = j.arrayExpression([]);
+		if (safeTraverse(p, [
+			"value",
+			"value",
+		]) && p.value.value.type === "ArrayExpression") {
+			arrExp = p.value.value;
+		}
+		value.forEach(val => {
+			addProperty(j, arrExp, null, val);
+		});
+		valForNode = arrExp;
+	} else if (
+		typeof value === "object" &&
+		!(value.__paths || value instanceof RegExp)
+	) {
+		let objectExp = j.objectExpression([]);
+		if (safeTraverse(p, [
```

Comment:
You'll have to check the entire key/val operation. Not sure if this is enough, let me test

Reply:
I've added some tests. Let me know if you run into any issues here.